pkg/admissionpolicy/utils.go

```diff
@@ -19,21 +19,28 @@ func hasPermissions(resource schema.GroupVersionResource, s checker.AuthChecker)
 // HasValidatingAdmissionPolicyPermission check if the admission controller has the required permissions to generate
 // Kubernetes ValidatingAdmissionPolicy
 func HasValidatingAdmissionPolicyPermission(s checker.AuthChecker) bool {
-	gvr := schema.GroupVersionResource{Group: "admissionregistration.k8s.io", Version: "v1beta1", Resource: "validatingadmissionpolicies"}
-	return hasPermissions(gvr, s)
+	gvr1 := schema.GroupVersionResource{Group: "admissionregistration.k8s.io", Version: "v1", Resource: "validatingadmissionpolicies"}
+	gvrbeta1 := schema.GroupVersionResource{Group: "admissionregistration.k8s.io", Version: "v1beta1", Resource: "validatingadmissionpolicies"}
+	return hasPermissions(gvr1, s) || hasPermissions(gvrbeta1, s)
 }
 
 // HasValidatingAdmissionPolicyBindingPermission check if the admission controller has the required permissions to generate
 // Kubernetes ValidatingAdmissionPolicyBinding
 func HasValidatingAdmissionPolicyBindingPermission(s checker.AuthChecker) bool {
-	gvr := schema.GroupVersionResource{Group: "admissionregistration.k8s.io", Version: "v1beta1", Resource: "validatingadmissionpolicybindings"}
-	return hasPermissions(gvr, s)
+	gvr1 := schema.GroupVersionResource{Group: "admissionregistration.k8s.io", Version: "v1", Resource: "validatingadmissionpolicybindings"}
+	gvrbeta1 := schema.GroupVersionResource{Group: "admissionregistration.k8s.io", Version: "v1beta1", Resource: "validatingadmissionpolicybindings"}
```

Comment:
No need to check permissions for v1beta1 as Kyverno 1.14 drops the support of the beta version.